Frontend/android-native/mobility-app/src/main/java/in/juspay/mobility/app/OverlayMessagingService.java

```diff
@@ -157,7 +182,63 @@ private void setDataToViews(String bundle) {
             buttonOk.setVisibility(buttonOkVisibility ? View.VISIBLE : View.GONE);
             buttonCancel.setVisibility(buttonCancelVisibility ? View.VISIBLE : View.GONE);
             imageView.setVisibility(imageVisibility ? View.VISIBLE : View.GONE);
+            JSONArray mediaViews = data.has("socialMediaLinks") ? data.getJSONArray("socialMediaLinks") : null;
```

Comment:
Add `mediaViews` properties in a different function, any exception in this part shouldn't block popup, Since backend is also using this so Please check backward compatibility .